R/data-mask.R

```diff
@@ -41,7 +41,16 @@ DataMask <- R6Class("DataMask",
         }
       }
 
-      env_bind_active(private$bindings, !!!set_names(map(seq_len(ncol(data)), binding_fn), names(data)))
+      bindings <- set_names(map(seq_len(ncol(data)), binding_fn), names(data))
+      env_bind_active(private$bindings, !!!bindings)
+
+      different <- which(chr_unserialise_unicode(names(bindings)) != env_names(private$bindings))
```

Comment:
Perhaps:

```suggestion
      workaround_bindings <- set_names(bindings, chr_unserialise_unicode(names(bindings)))
      different <- (names(workaround_bindings) != names(bindings))
      env_bind_active(private$bindings, !!!bindings[different])
```

and remove the loop?

Reply:
(Edited.)

Reply:
I don't think this works, because of how `env_bind_active()` works. Looping and effectively calling `makeActiveBinding()` was on purpose here